watsong/static/js/webapp.js

```diff
@@ -33,67 +33,67 @@ function getCoord(event) {
  * @param {string} dialName is the dial name to be sent to the backend
  */
 function initializeDial(elem, dialName) {
-    var currentAngle = 0, prevAngle = 0, initialAngle = 0, center = getCenter(elem);
+  var currentAngle = 0, prevAngle = 0, initialAngle = 0, center = getCenter(elem);
```

Comment:
let is better style than var, which is mainly used in older js code. https://google.github.io/styleguide/jsguide.html#features-use-const-and-let